src/fieldlist_SAMUNICON.jsonc

```diff
@@ -0,0 +1,72 @@
+// All text to the right of an unquoted "//" is a comment and ignored, as well
```

Comment:
Please move this file the diagnostics/albedofb directory for reference. If the existing CMIP fieldlist requires additional variables for the POD to run with the CMIP convention, I will add them.